packages/features/tasker/tasks/triggerNoShow/common.ts

```diff
@@ -109,19 +110,33 @@ export function calculateMaxStartTime(startTime: Date, time: number, timeUnit: T
     .unix();
 }
 
-export function checkIfUserJoinedTheCall(userId: number, allParticipants: Participants): boolean {
-  return allParticipants.some(
-    (participant) => participant.user_id && parseInt(participant.user_id) === userId
-  );
+export function checkIfUserOrGuestJoinedTheCall(
+  email: string,
+  allParticipants: ParticipantsWithEmail
+): boolean {
+  return allParticipants.some((participant) => participant.email && participant.email === email);
```

Comment:
I think for hosts we must check by userId only as otherwise, guest can provide organizer's email and be considered as organizer wrt no-show